pyctcdecode/decoder.py

```diff
@@ -665,6 +668,65 @@ def decode_batch(
         decoded_text_list: List[str] = pool.map(p_decode, logits_list)
         return decoded_text_list
 
+    def save_to_dir(self, filepath: str) -> None:
+        """Save a decoder to a directory."""
+        alphabet_path = os.path.join(filepath, self._ALPHABET_SERIALIZED_FILENAME)
+        with open(alphabet_path, "wb") as fi:
+            pickle.dump(self._alphabet, fi)  # nosec
```

Comment:
we could decide to serialize this to plain text- the alphabet is just a list of labels and whether its BPE. Looking for some comments